src/components/DeclareIncidentForm/DeclareIncidentFormSlice.js

```diff
@@ -0,0 +1,43 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  incidentName: '',
+  incidentType: '',
+  incidentDate: '',
+  incidentSummary: ''
+}
+
+export const slice = createSlice({
```

Comment:
Will it become a problem later with many slice files to name this function slice? Possibly more detail using naming conventions.